src/main/java/com/karol/kindergartenmanagementsystem/dto/SignInRequest.java

```diff
@@ -0,0 +1,25 @@
+package com.karol.kindergartenmanagementsystem.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SignInRequest {
+    @NotNull(message = "Email is mandatory")
```

Comment:
"Email is mandatory" if the same message is in many annotations, you should better extract it and make it a constant or enum